.changeset/afraid-colts-obey.md

```diff
@@ -0,0 +1,120 @@
+---
+'@keystone-ui/button': patch
+'@keystone-ui/core': patch
+'@keystone-ui/fields': patch
+'@keystone-ui/icons': patch
+'@keystone-ui/loading': patch
+'@keystone-ui/modals': patch
+'@keystone-ui/notice': patch
+'@keystone-ui/options': patch
+'@keystone-ui/pill': patch
+'@keystone-ui/popover': patch
+'@keystone-ui/segmented-control': patch
+'@keystone-ui/toast': patch
+'@keystone-ui/tooltip': patch
+'@keystone-ui/website': patch
+'@keystone-next/website': patch
+'@keystone-next/example-auth': patch
+'@keystone-next/app-basic': patch
+'@keystone-next/example-ecommerce': patch
+'@keystone-next/example-roles': patch
+'@keystone-next/example-todo': patch
+'@keystone-next/demo-project-blog-legacy': patch
+'@keystone-next/demo-custom-fields-legacy': patch
+'@keystone-next/demo-project-meetup-legacy': patch
+'@keystone-next/demo-project-todo-legacy': patch
+'@keystone-next/demo-project-todo-knex-legacy': patch
+'@keystone-next/admin-ui': patch
+'@keystone-next/admin-ui-utils': patch
+'@keystone-next/auth': patch
+'@keystone-next/cloudinary': patch
+'@keystone-next/document-renderer': patch
+'@keystone-next/fields': patch
+'@keystone-next/fields-document': patch
+'@keystone-next/keystone': patch
+'@keystone-next/session-store-redis': patch
+'@keystone-next/types': patch
+'@keystone-next/access-control-legacy': patch
+'@keystone-next/adapter-knex-legacy': patch
+'@keystone-next/adapter-mongoose-legacy': patch
+'@keystone-next/adapter-prisma-legacy': patch
+'@keystone-next/apollo-helpers-legacy': patch
+'@keystone-next/app-admin-ui-legacy': patch
+'@keystone-next/app-graphql-legacy': patch
+'@keystone-next/app-graphql-playground-legacy': patch
+'@keystone-next/app-next-legacy': patch
+'@keystone-next/app-nuxt-legacy': patch
+'@keystone-next/app-schema-router-legacy': patch
+'@keystone-next/app-static-legacy': patch
+'@keystone-next/app-version-legacy': patch
+'@arch-ui/docs': patch
```

Comment:
Having changesets for these packages while they're private and depended upon by other packages will result in broken packages on npm since other things will depend on versions that don't exist on npm.